client/state/selectors/get-current-plan-term.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Internal dependencies
+ */
+
+import { getPlan } from 'lib/plans';
+import { getSitePlan } from 'state/sites/selectors';
+
+/**
+ * Returns term of the active plan for given siteId, e.g. TERM_MONTHLY
+ *
+ * @param {Object} state Current state
+ * @param {Number} siteId Site ID
+ * @return {string} Human-readable interval type
```

Comment:
Return type should be capitalized. Also, the selector may return a `null` value, so it might be a good idea to add that to the return type (by prepending a question mark to the return type).